tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CancellationTests.cs

```diff
@@ -28,7 +21,7 @@ namespace Microsoft.Identity.Test.Integration.HeadlessTests
     [TestClass]
     public class CancellationTests
     {
-        [TestMethod]
+        [RunOn(TargetFrameworks.NetCore)]
```

Comment:
I'm thinking is there a better approach? Most tests should run on NetCore, we'll be adding a lot of these attributes. Maybe move all tests to Integration.NetCore project and then just reference a few files in .NetFx project with RunOn attribute?